report/string_set.go

```diff
@@ -54,33 +54,35 @@ func (s StringSet) Intersection(b StringSet) StringSet {
 // Add adds the strings to the StringSet. Add is the only valid way to grow a
 // StringSet. Add returns the StringSet to enable chaining.
 func (s StringSet) Add(strs ...string) StringSet {
+	r := s.Copy()
```

Comment:
OK, I will remove it. Maybe we should just rename Add/Remove to UnsafeMutableAdd/UnsafeMutableRemove to avoid usage in other contexts.